src/app/components/mnoe-tasks/mnoe-tasks.less

```diff
@@ -17,13 +17,22 @@ mnoe-tasks {
 
       & > .refresh-tasks {
         display: inline-block;
-        padding: 3px;
-        margin: 2px 2px 2px 7px;
+        padding: 5px;
+        margin: 2px 2px 2px 8px;
         color: @mno-tasks-column-sort-icon-active-color;
         &:hover {
           color: darken(@mno-tasks-column-sort-icon-active-color, 15%);
         }
+        &.disabled {
+          opacity: 0.5;
+          pointer-events: none;
```

Comment:
I would not recommend using `pointer-events: none`, it does not give the user any feedback and may feel like the page is frozen.

I recommend using `pointer: progress` or `pointer: not-allowed`. Here are the available pointer style examples: https://www.w3schools.com/cssref/tryit.asp?filename=trycss_cursor.